storage/testonly/admin_storage_tester.go

```diff
@@ -64,9 +64,9 @@ Ws9xezgQPrg96YGsFrF6KYG68iqyHDlQ+4FWuKfGKXHn3ooVtB/pfawb5Q==
 -----END PUBLIC KEY-----`
 )
 
-// mustMarshalAny panics if ptypes.MarshalAny fails.
+// mustMarshalAny panics if anypb.New(proto.MessageV2(p)) fails.
```

Comment:
Nit: comment could just say panics if it doesn't marshal, rather than the API involved.

Reply:
I think that was find and replace being too aggressive. Don't tell people how I live.